src/staticdata.c

```diff
@@ -1896,7 +1896,7 @@ static uint32_t write_gvars(jl_serializer_state *s, arraylist_t *globals, arrayl
         uintptr_t reloc = get_reloc_for_item(item, 0);
         write_reloc_t(s->gvar_record, reloc);
     }
-    return globals->len + 1;
+    return globals->len;
```

Comment:
```suggestion
    return globals->len + 1;
```

What motivated this change? This sets `external_fns_begin` https://github.com/JuliaLang/julia/blob/67938c02c7598850413b71b1b9aa159b6c20a33a/src/staticdata.c#L2389 which needs to point to one past `globals->len` 

https://github.com/JuliaLang/julia/blob/67938c02c7598850413b71b1b9aa159b6c20a33a/src/staticdata.c#L1917